pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -109,9 +109,48 @@ func (p *staticPolicy) Name() string {
 }
 
 func (p *staticPolicy) Start(s state.State) {
-	// Configure the shared pool to include all detected CPU IDs.
-	allCPUs := p.topology.CPUDetails.CPUs()
-	s.SetDefaultCPUSet(allCPUs)
+	// flag to indicate whether ignore state from file and start fresh state
+	initializeNewEmptyState := false
```

Comment:
Could we do the same kind of refactor as the other PR? Last time we moved state validation stuff to a helper function that returns error and got rid of a similar unrolled fold.

Reply:
ok